packages/main/src/types/ToolbarPriority.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Defines the priority of the toolbar item to go inside overflow popover.
+ *
+ * @readonly
+ * @enum {string}
+ * @public
+ * @type {string}
+ * @author SAP SE
+ * @alias sap.ui.webc.main.types.ToolbarPriority
+ */
+enum ToolbarPriority {
+
+	/**
+	 * @public
+	 * @type {Default}
+	 */
+	 Default = "Default",
+	/**
+	 * When set, the button will be always part of the overflow part of ui5-toolbar.
+
+	 * @public
+	 * @type {Never}
+	 */
+	Never = "Never",
+
+	/**
+	 * @public
+	 * @type {Always}
+	 */
+	Always = "Always",
```

Comment:
Maybe, we need to discuss couple of things in API discussion (@vladitasev ):

- the values: between Always, Never as are currently suggested in the PR, or maybe AlwaysOverflow, NeverOverflow

- the enum name: I find ToolbarPriority is not self-explanatory as well, maybe ToolbarOverflowPriority

Reply:
Yeah, or even `ToolbarItemOverflowPriority` or `ToolbarItemOverflowBehavior`